engines/sci/parser/vocabulary.cpp

```diff
@@ -288,12 +288,12 @@ bool Vocabulary::loadAltInputs() {
 		AltInput t;
 		t._input = data;
 
-		uint32 l = strlen(data);
+		uint32 l = Common::strnlen(data, data_end - data);
```

Comment:
Looks like I managed to miss quite a few places using strlen, not sure what happened there. I’ll get the other ones converted too and update this PR. I understand the rationale about wanting to error out too, I’m happy to make that change and see what happens.